packages/core/data/oot/world_mq/water_temple_mq.yml

```diff
@@ -21,22 +21,53 @@
     "MQ Water Temple Central Pillar Chest": "can_play_time && can_use_din && can_hookshot && has_iron_boots && has_tunic_zora_strict"
     "MQ Water Temple GS Lizalfos Hallway": "event(WATER_LEVEL_LOW) && can_use_din"
     "MQ Water Temple GS High Water Changer": "event(WATER_LEVEL_LOW) && can_longshot"
+    "MQ Water Temple Pot Ruto 1": "has_iron_boots && has_tunic_zora && can_hookshot"
+    "MQ Water Temple Pot Ruto 2": "has_iron_boots && has_tunic_zora && can_hookshot"
+    "MQ Water Temple Pot Storage Room 1": "has_iron_boots && has_tunic_zora && can_hookshot"
+    "MQ Water Temple Pot Storage Room 2": "has_iron_boots && has_tunic_zora && can_hookshot"
+    "MQ Water Temple Pot Storage Room 3": "has_iron_boots && has_tunic_zora && can_hookshot"
+    "MQ Water Temple Pot Twisted Room Entrance": "has_iron_boots && has_tunic_zora && can_hookshot"
+    "MQ Water Temple Pot Twisted Room Right 1": "has_iron_boots && has_tunic_zora && can_hookshot"
+    "MQ Water Temple Pot Twisted Room Right 2": "has_iron_boots && has_tunic_zora && can_hookshot"
```

Comment:
These three pots can use WATER_LEVEL_LOW and Hover Boots as well.

Reply:
fixed.